src/test/java/org/opensearch/geospatial/GeospatialObjectBuilder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial;
+
+import java.util.Map;
+
+import org.json.JSONObject;
+import org.opensearch.geospatial.geojson.Feature;
+
+public class GeospatialObjectBuilder {
```

Comment:
Why not use builder pattern for Geospatial object?

Reply:
Sorry, i was not clear. I updated the description. This is ObjectBuilder class for test. I add this so that both unit test and integration test can use this for input.